src/app/screens/BookDetail/components/CommentList/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { View, Text, FlatList, TouchableOpacity } from 'react-native';
+
+import Comment from './components/Comment';
+import styles from './styles';
+
+class CommentList extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { isViewedAll: false };
+    }
```

Comment:
we try to never use constructor? just add a line like:
`state = { isViewedAll: false }`